types/index.d.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference types="node" />
+
+import { ArcHTTP } from "./http";
+import { ArcStatic } from "./static";
+import { ArcWebSocket } from "./ws";
+import { ArcEvents, ArcQueues } from "./events";
+import { ArcTables } from "./tables";
+
+type ArcServices = () => Promise<Record<string, any>>;
```

Comment:
Would be nice to add more detail here, but I'm not sure what the full service discovery response from SSM would look like.